apps/console/src/features/identity-providers/components/wizards/enterprise-idp-create-wizard.tsx

```diff
@@ -691,51 +638,62 @@ export const EnterpriseIDPCreateWizard: FC<EnterpriseIDPCreateWizardProps> = (
             </Grid>
             <Divider hidden/>
             { (selectedProtocol === "oidc" && selectedCertInputType === "jwks") && (
-                <Field.Input
-                    ariaLabel="JWKS endpoint URL"
-                    inputType="url"
-                    name="jwks_endpoint"
-                    label="JWKS endpoint URL"
-                    required={ false }
-                    maxLength={ JWKS_URL_LENGTH.max }
-                    minLength={ JWKS_URL_LENGTH.min }
-                    width={ 15 }
-                    initialValue={ EMPTY_STRING }
-                    placeholder="Enter JWKS endpoint URL"
-                    data-testid={ `${ testId }-form-wizard-oidc-jwks-endpoint-url` }
-                />
+                <>
+                    <Field.Input
+                        ariaLabel="JWKS endpoint URL"
+                        inputType="url"
+                        name="jwks_endpoint"
+                        label="JWKS endpoint URL"
+                        required={ false }
+                        maxLength={ JWKS_URL_LENGTH.max }
+                        minLength={ JWKS_URL_LENGTH.min }
+                        width={ 15 }
+                        initialValue={ EMPTY_STRING }
+                        placeholder="Enter JWKS endpoint URL"
+                        data-testid={ `${ testId }-form-wizard-oidc-jwks-endpoint-url` }
+                    />
+                    <Hint>
+                        Asgardeo will use this URL to obtain keys to verify the signed
+                        responses from your external IdP
+                    </Hint>
```

Comment:
The text should be retrieved from i18n module. Please move them to language files.